Update maia-chat.md

RoelandSalij opened the pull request:
Added context for Maia in the Mendix Portal

RoelandSalij committed:
Update maia-chat.md

Added context for Maia Portal

RoelandSalij commented:
I added some context for the Maia Chat page.
Maia (Chat) is not a Studio Pro only concept, should we consider to take it out the Studio Pro menu/context?

MariaShaposhnikova commented:
@RoelandSalij , I would separate it into two documents, one for the Mendix Portal and one for Studio Pro and add cross-references for each other. Is this what you mean by taking out of SP context?

MarkvanMents commented:
I think splitting it into two documents is a good idea. We could add something to this document (https://docs.mendix.com/developerportal/global-navigation/) and then link to [Maia Chat](https://docs.mendix.com/refguide/maia-chat/) for more detail? And then ensure that Maia Chat is generic enough to cover both.
We need to wait until it is made GA (next Thursday?) before publishing this.

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -12,27 +12,27 @@ aliases:
 
 {{% alert color="info" %}}
-MendixChat was rebranded to Maia Chat and released in GA in Studio Pro 10.12.0.
+MendixChat was rebranded to Maia Chat and released in GA in Studio Pro 10.12.0 and the Mendix Portal.
```
```suggestion
MendixChat was rebranded to Maia Chat and released in GA in Studio Pro 10.12.0. It is also available in the Mendix Portal.
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -12,27 +12,27 @@ aliases:
 
 {{% alert color="info" %}}
-To use Maia Chat, internet connection and signing in to Studio Pro are required.
+To use Maia Chat, internet connection and signing in to Studio Pro or the Mendix Portal are required.
```
```suggestion
To use Maia Chat, an internet connection and signing in to Studio Pro or the Mendix Portal are required.
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -12,27 +12,27 @@ aliases:
 {{% /alert %}}
 
-Maia Chat is a built-in chat interface powered by Generative AI in Studio Pro. Developers can ask questions about app development in Mendix, including how to apply concepts, best practices, and development patterns. It is built based on large language models (LLMs), using data from various resources, such as [Mendix Docs](/), [Mendix Community](https://community.mendix.com/), and [Mendix Academy] ...
+Maia Chat is a built-in chat interface powered by Generative AI in the Mendix Platform . Developers can ask questions about app development in Mendix, including how to apply concepts, best practices, and development patterns. It is built based on large language models (LLMs), using data from various resources, such as [Mendix Docs](/), [Mendix Community](https://community.mendix.com/), and [Mendi ...
```
```suggestion
Maia Chat is a built-in chat interface in the Mendix Platform powered by Generative AI. Developers can ask questions about all aspects of Mendix, including how to apply concepts, best practices, governance tools, and development patterns. It is based on large language models (LLMs) using data from various resources, such as [Mendix Docs](/), [Mendix Community](https://community.mendix.com/), and [Mendix Academy](https://academy.mendix.com/).
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -12,27 +12,27 @@ aliases:
 
-* You can click the **Maia** ({{% icon name="sparkles" %}}) icon at the top right bar of Studio Pro
-* Or you can go to the **View** menu and click **Maia**
+* Studio Pro and Mendix Portal: You can click the **Maia** ({{% icon name="sparkles" %}}) icon at the top right bar.
+* Studio Pro: Or you can go to the **View** menu and click **Maia**
```
```suggestion
* In Studio Pro and Mendix Portal, click the **Maia** ({{% icon name="sparkles" %}}) icon to the right of the top bar.
* In Studio Pro go to the **View** menu and click **Maia**
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -12,27 +12,27 @@ aliases:
+* Studio Pro: Or you can go to the **View** menu and click **Maia**
 
-The chat interface appears at the right side of Studio Pro under the **Maia** tab:
+The chat interface appears at the right side of your context window or browser. In Studio Pro you can find the dock under the **Maia** tab.
```
What does "context window" mean?

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -45,8 +45,8 @@ The following steps present an example of how you can interact with Maia Chat by
     * An answer generated by Maia Chat
-    * A copy icon ({{% icon name="copy" %}}) which allows you to copy the given answer to a clipboard (this option was added in 10.7.0)
-    * Feedback icons
+    * Studio Pro: A copy icon ({{% icon name="copy" %}}) which allows you to copy the given answer to a clipboard (this option was added in 10.7.0)
+    * Studio Pro: Feedback icons
     * Top relevant references from various resources like Mendix documentation, Mendix Community, and Academy learning paths
```
```suggestion
    * Top relevant references from various resources like Mendix documentation, Mendix Community, and Academy learning paths
    
#### Studio Pro Features
    
Studio Pro has additional features which are indicated by the following:
    
    * A copy icon ({{% icon name="copy" %}}) which allows you to copy the given answer to a clipboard (this option was added in 10.7.0)
    * Feedback icons
    * New chat option
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -55,20 +55,23 @@ The following steps present an example of how you can interact with Maia Chat by
 
 3. To give your feedback on the given answer, click the thumbs up icon ({{% icon name="thumbs-up" %}}) or the thumbs down icon ({{% icon name="thumbs-down" %}}). 
```
```suggestion
1. Click the thumbs up icon ({{% icon name="thumbs-up" %}}) or the thumbs down icon ({{% icon name="thumbs-down" %}}) to give your feedback on an answer. 
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -55,20 +55,23 @@ The following steps present an example of how you can interact with Maia Chat by
 3. To give your feedback on the given answer, click the thumbs up icon ({{% icon name="thumbs-up" %}}) or the thumbs down icon ({{% icon name="thumbs-down" %}}). 
 
-4. To provide a more detailed feedback, click the add feedback icon ({{% icon name="notes-paper-edit" %}}). A detailed **Feedback** section shows below the given answer:
+4. In Studio Pro: To provide a more detailed feedback, click the add feedback icon ({{% icon name="notes-paper-edit" %}}). A detailed **Feedback** section shows below the given answer:
```
```suggestion
1. Click the add feedback icon ({{% icon name="notes-paper-edit" %}}) to provide a more detailed feedback. A detailed **Feedback** section shows below the answer:
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -55,20 +55,23 @@ The following steps present an example of how you can interact with Maia Chat by
+4. In Studio Pro: To provide a more detailed feedback, click the add feedback icon ({{% icon name="notes-paper-edit" %}}). A detailed **Feedback** section shows below the given answer:
 
     * You can rate the **correctness**, **completeness**, and **readability** of the provided answer.
     * You can add additional explanation text in the **Additional feedback** text box if you want to. 
```
```suggestion
    * You can also provide **Additional feedback** if you want to. 
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -55,20 +55,23 @@ The following steps present an example of how you can interact with Maia Chat by
         {{< figure src="/attachments/refguide/modeling/mendix-ai-assistance/maia-chat/feedback-section.png" width="250px" >}} 
 
-5. Click the **Submit feedback** button to send your feedback. Afterwards, you will be brought back to the main chat interface where you can continue asking new questions.
+5. In Studio Pro: Click the **Submit feedback** button to send your feedback. Afterwards, you will be brought back to the main chat interface where you can continue asking new questions.
```
```suggestion
1. Click the **Submit feedback** button to submit your feedback. Afterwards, you will be brought back to the main chat interface where you can continue asking questions.
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -55,20 +55,23 @@ The following steps present an example of how you can interact with Maia Chat by
+5. In Studio Pro: Click the **Submit feedback** button to send your feedback. Afterwards, you will be brought back to the main chat interface where you can continue asking new questions.
 
-6. If you want to clear the messages to start a new conversation in a clean state, navigate to the top of the interface and click **New chat** (this option was added in 10.7.0).
+6. In Studio Pro: If you want to clear the messages to start a new conversation in a clean state, navigate to the top of the interface and click **New chat** (this option was added in 10.7.0).
```
```suggestion
1. Click **New chat** at beginning of your chat if you want to clear the messages and start a new conversation without reference to your current chat (this option was added in 10.7.0).
```

MarkvanMents replied:
Was this really added in 10.7.0, two lines down we say that follow-up questions were only introduced in 10.8.0, so the **New Chat** options wouldn't have been necessary.

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -55,20 +55,23 @@ The following steps present an example of how you can interact with Maia Chat by
 ### Asking Follow-Up Questions
 
-In Studio Pro 10.8.0 and above, Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing the following in the chat: *Can you summarize that?*.
+In Studio Pro 10.8.0 and above and the Mendix Portal, Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing the following in the chat: *Can you summarize that?*.
```
```suggestion
In Studio Pro 10.8.0 and above, and in Mendix Portal, Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing the following in the chat: *Can you summarize that?*.
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -55,20 +55,23 @@ The following steps present an example of how you can interact with Maia Chat by
-In Studio Pro 10.8.0 and above, Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing the following in the chat: *Can you summarize that?*.
+In Studio Pro 10.8.0 and above and the Mendix Portal, Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing the following in the chat: *Can you summarize that?*.
+
+### Conversation History
```
I think this should be a subsection of **Asking Follow-Up Questions** as it doesn't make sense unless Maia knows about your previous chat.
```suggestion
#### Conversation History

```

MarkvanMents reviewed (dismissed):
@MariaShaposhnikova 
I've made a few changes which, I think, make it easier to see the two different implementations side-by-side.
Will you (and @Yiyun333) review these and follow up where necessary?

MarkvanMents committed:
Add release note

RoelandSalij committed:
Update _index.md

Changes to reflect the extended accessibility of Maia Chat by adding it to the Mendix Portal

MariaShaposhnikova committed:
Apply suggestions from code review

Co-authored-by: Mark van Ments <35492184+MarkvanMents@users.noreply.github.com>

MariaShaposhnikova committed:
Apply suggestions from code review

Co-authored-by: Mark van Ments <35492184+MarkvanMents@users.noreply.github.com>

MarkvanMents committed:
Proofread mendix-ai-assistance doc

MarkvanMents committed:
Merge pull request #9018 from mendix/RoelandSalij-patch-2

Update mendix-ai-assistance/_index.md

MariaShaposhnikova committed:
Fix a list

MariaShaposhnikova committed:
Separate Maia into SP and Mx Portal docs

MariaShaposhnikova commented on content/en/docs/refguide/modeling/mendix-ai-assistance/_index.md:
```diff
@@ -17,14 +17,13 @@ For information on Mendix data storage policies and practices for Maia, see [Mai
 
-* **Maia Chat** (released in GA in Studio Pro 10.12.0) – a built-in chat interface powered by Generative AI in Studio Pro. It answers questions about app development in Mendix, including how to apply concepts, best practices, and development patterns. For more information, see [Maia Chat](/refguide/maia-chat/).
-
+* **Maia Chat** (generally available in Studio Pro 10.12.0 and the Mendix Portal) – a built-in chat interface powered by Generative AI in Studio Pro. It answers questions about app development in Mendix, including how to apply concepts, best practices, and development patterns. For more information, see [Maia Chat](/refguide/maia-chat/). 
```
I've added Maia Chat back to the list rather than having it as a separate section, because Maia Chat is part of Maia, and not a completely separate thing as it may seem if we place it as a top-level section.

MarkvanMents replied:
I think we can remove the Mendix Portal from here as we are now just talking about Studio Pro.
```suggestion
* **Maia Chat** (generally available in Studio Pro 10.12.0) – a built-in chat interface powered by Generative AI in Studio Pro. It answers questions about app development in Mendix, including how to apply concepts, best practices, and development patterns. For more information, see [Maia Chat](/refguide/maia-chat/). 
```

MariaShaposhnikova commented on content/en/docs/developerportal/global-navigation.md:
```diff
@@ -54,3 +54,43 @@ To stop receiving notifications when new feedback is submitted for a specific ap
+
+Mendix AI Assistance (Maia) refers to Mendix Platform capabilities that leverage [artificial intelligence (AI)](https://www.mendix.com/glossary/artificial-intelligence-ai/) and [machine learning (ML)](https://www.mendix.com/glossary/machine-learning/) to assist developers in application development (via AI-assisted development (AIAD)). Maia is designed to help development teams in modeling and de ...
+
+Below are the Maia capabilities that are available in Mendix Portal:
```
@RoelandSalij , Do we have a screenshot to add to this doc?

RoelandSalij replied:
Yes, 
Updated one without the beta label:
<img width="472" alt="image" src="https://github.com/user-attachments/assets/f7f0ab1e-1c21-4e91-9e5f-90bae179367f" />

MariaShaposhnikova replied:
Thank you! I've used a slightly different one with a bit more information on it.

MariaShaposhnikova commented on content/en/docs/developerportal/global-navigation.md:
```diff
@@ -54,3 +54,43 @@ To stop receiving notifications when new feedback is submitted for a specific ap
+
+Below are the Maia capabilities that are available in Mendix Portal:
+
+* **Maia Chat**  – a built-in chat interface, for more information, see the [Maia Chat](#maia-chat) section below.
```
Added chat to the list as it is part of Maia.

MarkvanMents commented on content/en/docs/developerportal/global-navigation.md:
```diff
@@ -54,3 +54,43 @@ To stop receiving notifications when new feedback is submitted for a specific ap
+
+## Maia in Mendix Portal {#maia-mx-portal}
+
+Mendix AI Assistance (Maia) refers to Mendix Platform capabilities that leverage [artificial intelligence (AI)](https://www.mendix.com/glossary/artificial-intelligence-ai/) and [machine learning (ML)](https://www.mendix.com/glossary/machine-learning/) to assist developers in application development (via AI-assisted development (AIAD)). Maia is designed to help development teams in modeling and de ...
```
We never use "AI-assisted development" nor "AIAD" anywhere else in our documentation (except the Studio Pro 8.0 release notes 😀) and a search does not find it anywhere as a generally accepted industry abbreviation. I would remove it everywhere we have it unless we are going to use it somewhere. And, even then, I think we need a good reason to use it, since we now have MAIA as a term to use throughout the documentation. It is just repeating what the rest of the sentence says.

```suggestion
Mendix AI Assistance (Maia) refers to Mendix Platform capabilities that leverage [artificial intelligence (AI)](https://www.mendix.com/glossary/artificial-intelligence-ai/) and [machine learning (ML)](https://www.mendix.com/glossary/machine-learning/) to assist developers in application development. Maia is designed to help development teams in modeling and delivering Mendix applications faster, more consistently, and with higher quality. 
```

MariaShaposhnikova replied:
@MarkvanMents , that is a good point, thanks for checking the term and the abbreviation. I will remove it.

MarkvanMents commented on content/en/docs/developerportal/global-navigation.md:
```diff
@@ -54,3 +54,43 @@ To stop receiving notifications when new feedback is submitted for a specific ap
+### Maia Chat {#maia-chat}
+
+{{% alert color="info" %}}
+To use Maia Chat, an internet connection and signing in to Studio Pro or the Mendix Portal are required.
```
```suggestion
To use Maia Chat, you need to be signed in to Mendix and have an internet connection.
```

MarkvanMents commented on content/en/docs/developerportal/global-navigation.md:
```diff
@@ -54,3 +54,43 @@ To stop receiving notifications when new feedback is submitted for a specific ap
+Maia Chat does not use any project, customer, or company information. It only uses the data that you have entered in the Chat. Please do not enter any personal or sensitive data!
+{{% /alert %}}
+
+Maia Chat  is a built-in chat interface in the Mendix Platform powered by Generative AI. Developers can ask questions about all aspects of Mendix, including how to apply concepts, best practices, governance tools, and development patterns. Maia Chat is generally available in Studio Pro 10.12.0 and the Mendix Portal. For more information on Maia Chat in Studio Pro, see [Maia Chat](/refguide/maia-c ...
```
```suggestion
Maia Chat  is a built-in chat interface in the Mendix Platform powered by Generative AI. Developers can ask questions about all aspects of Mendix, including how to apply concepts, best practices, governance tools, and development patterns.

Maia Chat is also available in Studio Pro 10.12.0 and above. See [Maia Chat](/refguide/maia-chat/) for more information.
```

MarkvanMents commented on content/en/docs/developerportal/global-navigation.md:
```diff
@@ -54,3 +54,43 @@ To stop receiving notifications when new feedback is submitted for a specific ap
+
+#### Asking Follow-Up Questions
+
+In Mendix Portal, Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing the following in the chat: *Can you summarize that?*.
```
```suggestion
Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing *Can you summarize that?*.
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/_index.md:
```diff
@@ -41,10 +40,7 @@ Generators:
-
-* **Maia Rewrite** - an AI-assisted writing aid integrated within the Mendix Community posting interface, which helps you to ask properly-formatted questions that are more likely to be answered. For more information on how to use it, see the [Asking Your Question](/community-tools/mendix-community/#asking-question) section in *Mendix Community*.
-* **Maia Summarize** - an AI-assisted summarizing aid which condenses lengthy discussions for a question posted on Mendix Community into key points, making it easier for you to find solutions to your questions without having to plough through every single answer in the thread. For more information on how to use it, see the [Question Details](/community-tools/mendix-community/#question-details) se ...
+For more information on Maia in Mendix Portal, see the [Maia in Mendix Portal](/developerportal/global-navigation/#maia-mx-portal) section in *Global Navigation*.
```
```suggestion
Various Maia features are available in Mendix Portal. For more information, see the [Maia in Mendix Portal](/developerportal/global-navigation/#maia-mx-portal) section in *Global Navigation*.
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -12,27 +12,27 @@ aliases:
 
 {{% alert color="info" %}}
-To use Maia Chat, internet connection and signing in to Studio Pro are required.
+To use Maia Chat, an internet connection and signing in to Studio Pro or the Mendix Portal are required.
```
```suggestion
To use Maia Chat, an internet connection and signing in to Studio Pro are required.
```

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -45,31 +45,43 @@ The following steps present an example of how you can interact with Maia Chat by
-5. Click the **Submit feedback** button to send your feedback. Afterwards, you will be brought back to the main chat interface where you can continue asking new questions.
+6. Click **New chat** at beginning of your chat if you want to clear the messages and start a new conversation without reference to your current chat (this option was added in Studio Pro 10.7.0).
+#### Studio Pro Features
 
-6. If you want to clear the messages to start a new conversation in a clean state, navigate to the top of the interface and click **New chat** (this option was added in 10.7.0).
+Maia Chat in Studio Pro has additional features which are indicated by the following:
```
I know you still need to tidy this up - just making a note here!

MarkvanMents commented on content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md:
```diff
@@ -45,31 +45,43 @@ The following steps present an example of how you can interact with Maia Chat by
 
+{{% alert color="info" %}}
+
+Maia Chat only keeps track of a conversation history within a session in Studio Pro.
```
Do we need to be clearer about this. I don't think we are talking about an active session (i.e. Studio Pro remembering who we are) in the way we normally talk about sessions in app development. So I would probably say something like

```suggestion
Maia Chat will lose track of the conversation history if you exit Studio Pro.
```

MarkvanMents reviewed (commented):
I've made a few comments, but I know that you were still working on this so please ignore/rewrite if they aren't in line with what you are planning to write.

MariaShaposhnikova committed:
Apply suggestions from code review

Co-authored-by: Mark van Ments <35492184+MarkvanMents@users.noreply.github.com>

MariaShaposhnikova committed:
Remove abbreviation and a term

MariaShaposhnikova commented on content/en/docs/releasenotes/developer-portal/_index.md:
```diff
@@ -19,6 +19,12 @@ To see the current status of the Mendix Portal, see [Mendix Status](https://stat
+
+#### New Features
+
+* [Mendix AI Assistance Chat (Maia Chat)](/refguide/maia-chat/), previously available in Studio Pro, is now accessible in Mendix Portal.
```
@RoelandSalij , do we need to mention other features (Rewrite and Summarize) as well?

RoelandSalij replied:
As far as I know, these are available for a while. So I don't see a need to mention them now.

MariaShaposhnikova committed:
Formatting

MariaShaposhnikova committed:
Merge branch 'RoelandSalij-patch-1' of https://github.com/mendix/docs into RoelandSalij-patch-1

MariaShaposhnikova committed:
Add an image and make smaller changes

MariaShaposhnikova committed:
Update the image

Karuna-Mendix commented:
Merging the PR as requested on the [#Docs](https://mendix.slack.com/archives/C63U54B6K/p1738830600566479) channel.
